Common/Data/Market/Tick.cs

```diff
@@ -80,16 +80,7 @@ public decimal LastPrice
             }
         }
 
-        // In Base Class: Last Trade Tick:
```

Comment:
Please leave these comments in for doc. I often have people ask "Where is the time value in the Tick class"? etc.